README.rst

```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
 
    $ kytos napps install kytos/of_core
 
-If you are going to install kytos-napps from source code, all napps will be
-installed by default (just remember you need to enable the ones you want
-running).
+######
+Events
+######
+
+******
+Listen
+******
+
+kytos/core.openflow.raw.in
+==========================
+  Handle a RawEvent and generate a kytos/core.messages.in.* events.
+
+Content
+=======
```

Comment:
I believe this should be one level below "kytos/core.openflow.raw.in" (like @diraol did in of_topology).